src/index.js

```diff
@@ -3,51 +3,71 @@ import { createFilter } from '@rollup/pluginutils'
 export default function css(options = {}) {
   const filter = createFilter(options.include || ['**/*.css'], options.exclude)
   const styles = {}
-  const order = []
   let dest = options.output
-  let changes = 0
+  let hasChanged = false
+  let prevIds = []
 
   return {
     name: 'css',
+    buildStart() {
+      hasChanged = false
```

Comment:
If a build takes 10s and you save an irrelevant file after 5s, `hasChanged` will be false at the moment that generateBundle takes place. I think this might introduce a race-condition. Does this change address a bug in the current version?

Maybe we should remove all change detection to make things simpler?

Reply:
That change is from my pull request. You raise a good point, I’ll think about what to do about this and update my PR